controls/sshd_spec.rb

```diff
@@ -77,7 +77,6 @@
     it { should exist }
     it { should be_file }
     it { should be_owned_by 'root' }
-    it { should be_grouped_into 'root' }
```

Comment:
Is it possible to set notifications so that when CI tests fail I would get notified?